pkg/tsdb/cloud-monitoring/converter/converter.go

```diff
@@ -566,12 +564,16 @@ func readScalar(iter *jsonitere.Iterator) backend.DataResponse {
 		Custom: resultTypeToCustomMeta("scalar"),
 	}
 
+	if dataPlane {
```

Comment:
Why is this change being made?

Reply:
https://github.com/grafana/grafana/pull/81491

Reply:
Thank you for clarifying! All good to merge 😊